### PR TITLE
Enabling Validation, Captcha, Translations, and Conditions Reference

### PR DESCRIPTION
**Pull request by justinchoi001**
CC @rbohl

### CODE REVIEW

File: docs/dxp/7.x/en/process-automation/forms/user-guide/advanced-forms-usage/translating-forms.md [hunk 1]
```diff
@@ -1,3 +1,39 @@
 # Translating Forms
 
-Coming soon!
+Forms can be translated to any locale that Liferay DXP supports.
+
+To add a translate a form:
```
Comment (jrhoun) [inline docs/dxp/7.x/en/process-automation/forms/user-guide/advanced-forms-usage/translating-forms.md:5]: > To add a translate a form:
Please review and revise.

File: docs/dxp/7.x/en/process-automation/forms/user-guide/advanced-forms-usage/translating-forms.md [hunk 2]
```diff
@@ -1,3 +1,39 @@
 # Translating Forms
 
-Coming soon!
+Forms can be translated to any locale that Liferay DXP supports.
```
Comment (jrhoun) [inline docs/dxp/7.x/en/process-automation/forms/user-guide/advanced-forms-usage/translating-forms.md:3]: > Forms can be translated to any locale that Liferay DXP supports.

More accurately: Forms can be be translated and presented in any language that is currently enabled on the current site.

**Comment by jrhoun** [comment]
Please review and re-send. Thanks!

File: docs/dxp/7.x/en/process-automation/forms/user-guide/advanced-forms-usage/translating-forms.md (hunk 1 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/process-automation/forms/user-guide/advanced-forms-usage/translating-forms.md:5]: Revised

File: docs/dxp/7.x/en/process-automation/forms/user-guide/advanced-forms-usage/translating-forms.md (hunk 2 above)
Comment (justinchoi001) [reply docs/dxp/7.x/en/process-automation/forms/user-guide/advanced-forms-usage/translating-forms.md:3]: Revised

File: docs/dxp/7.x/en/process-automation/forms/user-guide/advanced-forms-usage/enabling-captcha-on-form-submissions.md [hunk 3]
```diff
@@ -1,3 +1,23 @@
 # Enabling Captcha on Form Submissions
 
-Coming soon!
+[CAPTCHA](https://en.wikipedia.org/wiki/CAPTCHA) prevents a bot from submitting forms.
```
Comment (jrhoun) [inline docs/dxp/7.x/en/process-automation/forms/user-guide/advanced-forms-usage/enabling-captcha-on-form-submissions.md:3]: Please improve the introductory statement here: https://docs.magento.com/user-guide/stores/security-captcha.html gives a bit more as an example.

**Pull request closed without merging**